doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -41,14 +41,16 @@ Displaying
       An underscore means the usual name is printed.
 
 
-.. cmd:: About @qualid
+.. cmd:: About @smart_qualid {? @%{ {* @name } %} }
    :name: About
 
    This displays various information about the object
    denoted by :n:`@qualid`: its kind (module, constant, assumption, inductive,
    constructor, abbreviation, …), long name, type, implicit arguments and
    argument scopes. It does not print the body of definitions or proofs.
 
+   .. todo: what does this variant add?
+
    .. cmdv:: About @qualid\@@name
```

Comment:
That's not even correct syntax AFAICT. However, it could be interesting to use the `cmdv` to desribe the variant where the `@{ }` syntax is used (I personally didn't know it existed).